src/goInstallTools.ts

```diff
@@ -110,7 +110,8 @@ function installTools(goVersion: SemVersion, missing?: string[]) {
 
 	missing.reduce((res: Promise<string[]>, tool: string) => {
 		return res.then(sofar => new Promise<string[]>((resolve, reject) => {
-			cp.exec('go get -u -v ' + tools[tool], { env: process.env }, (err, stdout, stderr) => {
+			let goRuntimePath = getGoRuntimePath();
```

Comment:
You would probably only need to lookup the goRuntimePath once during the install phase so to improve performance move it above the reduce function.